Cargo.toml

```diff
@@ -28,10 +28,13 @@ std-fs-io = ["casper-types/std-fs-io"]
 [dependencies]
 async-trait = { version = "0.1.74", optional = true }
 base16 = "0.2.1"
+base64 = "0.21.5"
```

Comment:
Could we please restrict this import to the minimum required ?

base64 = { version = "0.22.1", default-features = false }

or maybe

base64 = { version = "0.22.1", default-features = false, features = ["alloc"] }